Kaggle instructions: Import notebooks via download button

vasudev-sharma opened the pull request:
## Motivation 🚀
The Kaggle instructions (https://github.com/NeuromatchAcademy/course-content/pull/594) written earlier were difficult to follow through as they were missing the url of notebook. So, we decided rather than importing the files via the "url", we will straightaway "upload" the "downloaded" notebooks to Kaggle.

## What it does? 🤔
To upload the notebooks on Kaggle, instructions for downloading the notebook file on website (https://compneuro.neuromatch.io/intro.html) and then uploading them to Kaggle kernel are added with some images in this PR.

vasudev-sharma commented:
@ebatty This PR is up for review.

ebatty reviewed (changes requested):
1) Could you move this into tutorials/Technical Help
2) Could you add above gpu step that this is deep learning specific. They will not need gpus for comp neuro tutorials, only for the deep learning course
.

vasudev-sharma commented:
> 1. Could you move this into tutorials/TechnicalHelp
> 2. Could you add above gpu step that this is deep learning specific. They will not need gpus for comp neuro tutorials, only for the deep learning course
>    .

I've addressed your comments. @ebatty could you re-review this PR?
